docs/docs/core/02-context.md

```diff
@@ -40,6 +40,10 @@ https://github.com/cosmos/cosmos-sdk/blob/v0.50.0-alpha.0/types/context.go#L41-L
 * **Priority:** The transaction priority, only relevant in `CheckTx`.
 * **KV `GasConfig`:** Enables applications to set a custom `GasConfig` for the `KVStore`.
 * **Transient KV `GasConfig`:** Enables applications to set a custom `GasConfig` for the transiant `KVStore`.
+* **StreamingManager:** The streamingManager field provides access to the streaming manager, which allows modules to subscribe to events emitted by the blockchain.**
```

Comment:
```suggestion
* **StreamingManager:** The streamingManager field provides access to the streaming manager, which allows modules to subscribe to events emitted by the blockchain.
```

I wouldn't word it as _events_  but state changes instead, as it may be easy to confuse with actual _events_.
Additionally, I think we should add a link to ADR 38 for more context.